tests/js/spec/utils/discover/charts.spec.tsx

```diff
@@ -0,0 +1,72 @@
+import {
```

Comment:
I've converted this test file to tsx to provide typing, lmk if thats okay :)

Reply:
Yep, totally okay 👍